### PR TITLE
Set up Firebase hosting

### PR DESCRIPTION
**Pull request by anyatokar**
**Change summary:**
1. Set up Firebase hosting
2. Simplified the `yarn build` script 💕
3. Deleted Docker deployment and related directories since they're no longer needed
4. Updated DEPLOYMENT doc with new steps

Here's Google's doc I used: https://firebase.google.com/docs/hosting/quickstart#initialize

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -18,21 +18,34 @@ gcloud config set project mfte-simple-92c08
 
 ## Deploying
 
-Make sure the app and docker containers are built:
```
Comment (anyatokar) [inline DEPLOYMENT.md:21]: DNPTL: @mmcknett do you think it's a good idea to get rid of everything associated with Docker/"old way" of deploying? That's what I did in this PR but open feedback!

File: public/index.html [hunk 2]
```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
```
Comment (anyatokar) [inline public/index.html:2]: DNPTL: On further thought, getting rid of the public folder completely was probably not the move 😂

File: DEPLOYMENT.md (hunk 1 above)
Comment (mmcknett) [reply DEPLOYMENT.md:21]: What does DNPTL stand for? I think getting rid of docker & nginx now is the right thing to do!

**Review by mmcknett — APPROVED** [review]
Awesome! I love that it was straightforward to make things simpler. 😄

File: public/index.html (hunk 2 above)
Comment (mmcknett) [reply public/index.html]: Yeah, you still need the root `div` for React to mount to something. 🙂

File: DEPLOYMENT.md (hunk 1 above)
Comment (anyatokar) [reply DEPLOYMENT.md:21]: @mmcknett "does not pertain to line" Thanks for asking. People use it at work and I sometimes assume I'm the only one not knowing an acronym!

**Pull request merged**